infra/scripts/migrate.sh

```
@@ -82,13 +85,13 @@
     print_status "Waiting for database to be ready..."
     
     # Check if we're running in Docker environment
-    if docker compose -f ./docker/docker-compose.yml ps mysql | grep -q "Up"; then
+    if docker compose --env-file $ENV_FILE -f $DOCKER_COMPOSE_FILE ps mysql | grep -q "Up"; then
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)